types/router.d.ts

```diff
@@ -1,4 +1,135 @@
-import Vue, { ComponentOptions, PluginFunction, AsyncComponent } from 'vue'
+import Vue, { VNode, ComponentOptions, PluginFunction, AsyncComponent } from 'vue' 
+
+export interface RouterLinkProps {
```

Comment:
can you move the type declarations to `RouterLink.d.ts` and `RouterView.d.ts`?

Reply:
done